tests/test_directives.py

```diff
@@ -155,59 +180,122 @@ def test_set_undefined_directive(self, setUp, available_directives_list, product
 
     def test_set_directives_item(self, setUp, available_directives_list, product_directive):
         directives = setUp(available_directives_list=available_directives_list)
-        directives[product_directive.name] = product_directive.default
-        assert directives[product_directive.name] == product_directive.default
+        directives['test'] = True
+        assert directives['test']
 
     def test_del_directive(self, setUp, available_directives_list, product_directive):
```

Comment:
```suggestion
    def test_del_directive(self, setUp, available_directives_list):
```

No need for that fixture now.